src/graph/two_player_graph.py

```diff
@@ -365,6 +365,7 @@ def __call__(self,
         :return: A concrete/active instance of the TwoPlayerGraph
         """
         self._instance = TwoPlayerGraph(graph_name, config_yaml, save_flag)
+        self._instance._graph = nx.MultiDiGraph(name=graph_name)
```

Comment:
NEVER ACCESS PRIVATE VARIABLES OUTSIDE A CLASS. 
`self._instance._graph = nx.MultiDiGraph(name=graph_name)` is already taken care of inside the constructor of `TwoPlayerGraph`